pkg/api/common/const.go

```diff
@@ -301,6 +301,7 @@ const (
 )
 
 const WindowsArtifactComponentName = "windowszip"
+const WindowsArtifactAzureStackComponentName = ""
```

Comment:
this const is the key of the map entry, please use a meaningful name like `windowszip-azs` or similar